content/en/actions/app_builder/saved_actions.md

```diff
@@ -0,0 +1,61 @@
+---
+title: "Save and Reuse Actions"
+description: Save and reuse an action and its parameters
+disable_toc: false
+disable_sidebar: false
+algolia:
+  tags: ['app builder', 'actions']
+type: documentation
+aliases:
+  - /app_builder/actions_catalog/action_blueprints/
+  - /service_management/app_builder/actions_catalog/saved_actions/
+  - /service_management/app_builder/actions/saved_actions/
+  - /service_management/app_builder/saved_actions
```

Comment:
Did these URLs exist before? If not, and if they're just copied from the Workflows version of this page, we probably don't need them because people won't be going to them 🙂